gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-webhook/src/main/resources/schemas/subscription/schema-form.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
⚠️ Beware, we decided to use subscriptions with plural form, so here the folder is not well named

Reply:
:heavy_check_mark: